src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/CategoryControllerTest.php

```diff
@@ -1436,26 +1439,28 @@ public function testDelete()
 
         $this->em->flush();
 
-        $client = $this->createAuthenticatedClient();
+        $categoryId = $category->getId();
+
+        $client = $this->client;
         $client->request(
             'DELETE',
-            '/api/categories/' . $category->getId()
+            '/api/categories/' . $categoryId
         );
 
         $this->assertHttpStatusCode(204, $client->getResponse());
 
-        $client = $this->createAuthenticatedClient();
+        $client = $this->client;
```

Comment:
This line would not be necessary, would it?

Reply:
Removed all appearance of `$client = $this->client;`